run-tests.sh

```diff
@@ -24,6 +24,6 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 pydocstyle invenio_workflows && \
-sphinx-build -qnNW docs docs/_build/html && \
+sphinx-build -qnN docs docs/_build/html && \
 python setup.py test && \
-sphinx-build -qnNW -b doctest docs docs/_build/doctest
+sphinx-build -qnN -b doctest docs docs/_build/doctest
```

Comment:
That error is due to the celery tasks issue that @jacquerie pointed out (inveniosoftware/troubleshooting#8), maybe we can pin celery only on the docs requirements.